src/Microsoft.Azure.WebJobs.ServiceBus/Config/ServiceBusConfiguration.cs

```diff
@@ -14,6 +16,18 @@ public class ServiceBusConfiguration
         private string _connectionString;
 
         /// <summary>
+        /// Constructs a new instance.
+        /// </summary>
+        public ServiceBusConfiguration()
+        {
+            MessageOptions = new OnMessageOptions
+            {
+                MaxConcurrentCalls = 5
```

Comment:
There isn't a corresponding setting for Queues. Concurrency is governed by Batch size and NewBatchThreshold.

There is a slight concern here that setting this to something other than 1 might regress people if they're relying on no concurrency. However, I think that is unlikely to be an issue and is outweighed by the benefit of giving people concurrency by default (which is what they would expect).

So we're looking for the right default number here.